tests/test_arrow_dataset.py

```diff
@@ -3553,3 +3554,32 @@ def test_task_text_classification_when_columns_removed(self):
         with Dataset.from_dict(data, info=info) as dset:
             with dset.map(lambda x: {"new_column": 0}, remove_columns=dset.column_names) as dset:
                 self.assertDictEqual(dset.features, features_after_map)
+
+
+def test_train_test_split_startify():
```

Comment:
Added tests for testing error messages inside `test_errors_train_test_split_stratify` function of `StratifiedTest` class. I also moved `test_train_test_split_startify` function inside same class so that all the functions testing stratified split comes under same class.